cmd/minikube/cmd/start.go

```diff
@@ -95,6 +95,7 @@ const (
 	keepContext           = "keep-context"
 	createMount           = "mount"
 	featureGates          = "feature-gates"
+	nodeName              = "node-name"
```

Comment:
I don't think we need a new flag for node-name, in minikube's terminology profile name should be same as node name